### ISSUE TITLE
im2txt: How to Calculate metrics from logits and targets

### ISSUE THREAD

**Issue by zBabar** [tensorflow/models#3790]
Hi.

Perhaps I am not very good at Tensorflow. Can anybody guide me how to calculate BLEU and other metric scores if we have target and logits like we have in code im2txt. Especially If it uses High Level API from tf.contrib.slim.learning.train instead of standard tensorflow session based approach.

**Comment by tensorflowbutler** [comment]
Thank you for your post. We noticed you have not filled out the following field in the issue template. Could you update them if they are relevant in your case, or leave them as N/A? Thanks.
What is the top-level directory of the model you are using
Have I written custom code
OS Platform and Distribution
TensorFlow installed from
TensorFlow version
Bazel version
CUDA/cuDNN version
GPU model and memory
Exact command to reproduce
